lib/project-view.coffee

```diff
@@ -10,6 +10,14 @@ module.exports = ProjectView =
       type: 'boolean'
       default: true
       description: 'Show the project path after project name in tree-view root.'
+    regexMatch:
+        type: 'string'
+        default: ""
+        description: 'Use a custom regex to transform the path'
+    rewrite:
```

Comment:
As we're going to perform a regex replace it might be more consistent if we name the setting `regexSubStr`